qchem/tests/test_active_space.py

```diff
@@ -20,17 +22,27 @@
     ],
 )
 def test_active_spaces(
-    mol_name, n_act_electrons, n_act_orbitals, docc_indices_ref, act_indices_ref
+    mol_name, n_act_electrons, n_act_orbitals, core_ref, active_ref
 ):
 
     r"""Test the correctness of the generated active spaces"""
 
-    docc_indices, active_indices = qchem.active_space(
-        mol_name, ref_dir, n_act_electrons, n_act_orbitals
+    molecule = MolecularData(filename=os.path.join(ref_dir, mol_name))
+
+    core, active = qchem.active_space(
+        molecule.n_electrons,
+        molecule.n_orbitals,
+        mult=molecule.multiplicity,
+        nact_els=n_act_electrons,
+        nact_orbs=n_act_orbitals
     )
 
-    assert docc_indices == docc_indices_ref
-    assert active_indices == act_indices_ref
+    # docc_indices, active_indices = qchem.active_space(
+    #     mol_name, ref_dir, n_act_electrons, n_act_orbitals
+    # )
```

Comment:
Is this necessary to keep?

Reply:
Thanks @soranjh , this is something I forgot to delete. Thanks!.